HRTool/src/main/java/app/controllers/ReportRestController.java

```diff
@@ -61,6 +58,18 @@
 	@Autowired
     private ServletContext servletContext;
 	
+	/**
+	 * Returns only the tag clouds that contain the given employee
+	 * @param extracted
+	 * @param toExtract
+	 * @param employee
+	 * @return
+	 */
+	private void extractClouds(Set<TagCloud> extracted, List<TagCloud> toExtract, Employee employee){
+		toExtract.stream().filter(tc -> tc.getEmployees().contains(employee)).forEach(tc -> extracted.add(tc));
+	}
+	
+	@SuppressWarnings({ "rawtypes", "unchecked" })
 	@RequestMapping("/cv")

```

Comment:
Is the CV generated for an employee? If so, this method should be in employee controller:
/employees/{id}/cv with a GET method.